packages/helix/helix.pacscript

```diff
@@ -0,0 +1,36 @@
+name="helix"
+version="22.08.1"
+repology=("project: helix")
+url="https://github.com/helix-editor/helix/archive/refs/tags/${version}.tar.gz"
+build_depends="cargo git"
+description="A post-modern modal text editor"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+hash="c9b184d53bab51e7a9fe81dc8c191a853177abba67ea408e968937f31b50b45a"
+
+prepare() {
+    true
+}
+
+build() {
+    cargo build -j$(nproc) --release --locked
+}
+
+install() {
+    sudo install -Dm755 target/release/hx -t "${pkgdir}/usr/lib/${name}"
+
+    wget -q "https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/${name}/hx"
+    sudo install -Dm755 hx -t "${pkgdir}/usr/bin"
```

Comment:
Couldn't this be replaced with a simple `echo "script"`?

Reply:
Sure, let me adjust that in a sec 😄

Reply:
Do you prefer me squashing this adjustment or just adding as another commit?

Reply:
Whatever works for you

Reply:
Done